Lib/unittest/case.py

```diff
@@ -227,7 +227,7 @@ class _AssertWarnsContext(_AssertRaisesBaseContext):
     def __enter__(self):
         # The __warningregistry__'s need to be in a pristine state for tests
         # to work properly.
-        for v in sys.modules.values():
+        for v in tuple(sys.modules.values()):
```

Comment:
Use `list()`. There is no benefit of using a tuple, but creating a list from a general iterable is a tiny bit faster.

`tuple(x)` is expanded to `tuple(list(x))` in most cases (if `x` is not a tuple or list).